app/models/form/sales/subsections/income_benefits_and_savings.rb

```diff
@@ -14,6 +14,7 @@ def pages
       Form::Sales::Pages::Buyer1Mortgage.new(nil, nil, self),
       Form::Sales::Pages::MortgageValueCheck.new("buyer_1_mortgage_value_check", nil, self, 1),
       Form::Sales::Pages::Buyer2Income.new(nil, nil, self),
+      Form::Sales::Pages::Buyer2IncomeValueCheck.new("buyer_2_income_value_check", nil, self),
```

Comment:
I'm not sure these ids are as clear as they could be, though this is a problem elsewhere too